routes/votings.js

```diff
@@ -42,7 +42,7 @@ router.patch('/:id', (request, response, next) => {
   const { id } = request.params
   const { isCanceled, first_movie_won } = request.body
 
-  if (!first_movie_won || !isCanceled) {
+  if (first_movie_won === undefined || isCanceled === undefined) {
```

Comment:
Isn't this the exact same? I like the first way you have written this a bit better.